test/extended/util/disruption/backend_sampler_tester.go

```diff
@@ -134,6 +134,12 @@ func (t *backendDisruptionTest) Test(f *framework.Framework, done <-chan struct{
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 
+	allowedDisruption, disruptionDetails, err := t.getAllowedDisruption(f)
+	if allowedDisruption == nil{
+		framework.Logf(fmt.Sprintf("Skipping: %s: No historical data", t.testName))
```

Comment:
I also tried calling `skipper.Skipf(....` here but this resulted in the entire upgrade disruption test being skipped.

Reply:
Yea the e2eskipper won't work here.  You would have to write a FrameworkSkipf in test/extended/util/disruption/disruption.go to generate a skipped test, which probably requires adding that functionality to TestSummaries.